### ISSUE TITLE
Uncaught Error: spawn ENOTDIR

### ISSUE THREAD

**Issue by dohzya** [AtomLinter/linter-scalac#19]
Hi,

I used the previous version (0.3.4) without any problem, but just after having updated this plugin to 1.0.0 (and the linter plugin to 1.3.1) I started seeing this error every time I save a file.

- I tried to set the scalac executable path without success (I checked the entered path and it's valid)
- I tried (just in case) restart atom but it didn't change the problem

I hope this report could help :-)

**Atom Version**: 1.0.4
**System**: Mac OS X 10.11
**Thrown From**: [linter-scalac](https://github.com/AtomLinter/linter-scalac) package, v1.0.0

### Stack Trace

Uncaught Error: spawn ENOTDIR

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:260

Error: spawn ENOTDIR
  at exports._errnoException (util.js:734:11)
  at ChildProcess.spawn (child_process.js:1145:11)
  at Object.exports.spawn (child_process.js:977:9)
  at BufferedProcess.module.exports.BufferedProcess.spawn (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:188:44)
  at new BufferedProcess (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:52:14)
  at /Users/evo/.atom/packages/linter-scalac/node_modules/atom-linter/lib/helpers.coffee:40:30
  at Object.module.exports.Helpers._exec (/Users/evo/.atom/packages/linter-scalac/node_modules/atom-linter/lib/helpers.coffee:22:16)
  at Object.module.exports.Helpers.exec (/Users/evo/.atom/packages/linter-scalac/node_modules/atom-linter/lib/helpers.coffee:13:13)
  at Object.provider.lint (/Users/evo/.atom/packages/linter-scalac/lib/linter-scalac.coffee:46:24)
  at Promise.then._this.emitter.emit.linter (/Users/evo/.atom/packages/linter/lib/linter-registry.coffee:55:22)
  at LinterRegistry.triggerLinter (/Users/evo/.atom/packages/linter/lib/linter-registry.coffee:54:16)
  at /Users/evo/.atom/packages/linter/lib/linter-registry.coffee:48:17
  at Array.map (native)
  at /Users/evo/.atom/packages/linter/lib/linter-registry.coffee:46:27

```

### Commands

```
     -1:33.9.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -1:31.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:25.9.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -1:25.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:39.7.0 pane:show-previous-item (atom-text-editor.editor.is-focused)
  9x -0:35.5.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:32 pane:show-previous-item (atom-text-editor.editor.mini.is-focused)
  2x -0:31.6.0 pane:show-next-item (atom-text-editor.editor.is-focused)
     -0:26.6.0 editor:select-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -0:26.4.0 core:select-right (atom-text-editor.editor.is-focused)
     -0:26 core:copy (atom-text-editor.editor.is-focused)
     -0:25.7.0 linter:set-bubble-transparent (atom-text-editor.editor.is-focused)
     -0:16.9.0 editor:delete-line (atom-text-editor.editor.is-focused)
     -0:16.5.0 core:save (atom-text-editor.editor.is-focused)
  2x -0:15.7.0 core:close (atom-text-editor.editor.is-focused)
  2x -0:12.7.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "followSymlinks": true,
    "disabledPackages": [
      "linter-cpplint"
    ],
    "themes": [
      "one-light-ui",
      "base16-syntax"
    ],
    "audioBeep": false,
    "ignoredNames": [
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "*.o",
      ".*~",
      "cscope*.out"
    ]
  },
  "linter-scalac": {
    "scalacExecutablePath": "/Users/evo/.jenv/shims/scalac"
  }
}
```

### Installed Packages

```coffee
# User
Stylus, v1.0.0
Sublime-Style-Column-Selection, v1.3.0
atom-beautify, v0.28.8
base16-syntax, v1.5.0
change-case, v0.6.0
clang-format, v1.15.0
color-picker, v2.0.11
ease-blink, v0.5.2
file-icons, v1.6.2
fold-functions, v0.2.1
git-log, v0.4.1
git-tab-status, v1.9.2
go-plus, v3.4.4
highlight-selected, v0.10.1
imdone-atom, v1.1.9
language-cmake, v0.1.4
language-diff, v0.3.0
language-gitignore, v0.2.0
language-scala, v1.1.3
last-cursor-position, v0.9.0
linter, v1.3.1
linter-clang, v3.1.1
linter-jshint, v1.1.4
linter-jsonlint, v0.1.4
linter-ruby, v0.1.6
linter-scalac, v1.0.0
linter-shellcheck, v0.0.8
linter-stylint, v2.0.0
local-history, v3.2.3
minimap, v4.12.2
minimap-bookmarks, v0.1.0
minimap-linter, v1.0.0
project-manager, v1.15.11
tab-control, v0.6.7
tasks, v2.4.0
tree-view-filter, v0.3.7
tree-view-git-status, v0.1.2

# Dev
No dev packages
```

**Comment by k2b6s9j** [comment]
Thanks for reporting. I'll fix this now.

**Comment by k2b6s9j** [comment]
Please reopen this issue if it continues.

**Comment by dohzya** [comment]
I confirm it's now working, thank you :-)

**Comment by k2b6s9j** [comment]
You are very welcome.
